components/tabs/stories/tabs.stories.js

```diff
@@ -104,7 +96,7 @@ export default {
 			},
 			{
 				label: "Tab 3",
```

Comment:
This wasn't there originally, but is there a way for users to see the label-only versions? Would that be maybe more work that it's worth?